kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/balance/presenter/BalancePresenter.java

```diff
@@ -36,22 +90,149 @@ private void updateBalance(Balance balance) {
 		if (balanceValue == 0) {
 			balanceString = BALANCE_ZERO_TEXT;
 		} else {
-			balanceString = StringUtil.getAmountFormatted(balanceValue);
+			balanceString = getAmountFormatted(balanceValue);
 		}
 		if (view != null) {
 			view.updateBalance(balanceString);
 		}
 	}
 
+	private int getType(OfferType offerType) {
+		switch (offerType) {
+			case SPEND:
+				return SPEND;
+			default:
```

Comment:
`default:` in the end for readbilty

Reply:
why default is earn?

Reply:
no particular reason.

Reply:
@OrenZak also annotate return with @OrderType